src/documentation/0001-node-introduction/index.md

```diff
@@ -59,7 +59,8 @@ Node.js has a fantastic [standard library](https://nodejs.org/api/), including f
 
 The `createServer()` method of `http` creates a new HTTP server and returns it.
 
-The server is set to listen on the specified port and host name. When the server is ready, the callback function is called, in this case informing us that the server is running.
+The server is set to listen on the specified port and host name. When you deploy your application to a Node-specific hosted environment, a PORT environment variable is provided on which your server can run. To access your application, your port property must be assigned the process.env.PORT value. To allow for both offline and hosted usage, we define the port as const port = 3000 || process.env.PORT. Note that you can use any port number of your choice if it is not currently in use by any other service. 
```

Comment:
```suggestion
The server is set to listen on the specified port and host name. When running the server a PORT environment variable can be provided if not it will default to `3000`. Note that you can use any port number of your choice if it is not currently in use by any other service or limited by the host environment.
```

Reply:
Myles's suggestion is missing some punctuation. I'd also get rid of "Note that". 

```suggestion
The server is set to listen on the specified port and host name. When running the server, a PORT environment variable can be provided. If not, it will default to `3000`. You can use any port number of your choice if it is not currently in use by any other service or limited by the host environment.
```